airflow/jobs/scheduler_job.py

```diff
@@ -1552,7 +1549,7 @@ def _execute_helper(self):
                 self.log.debug("Waiting for processors to finish since we're using sqlite")
                 self.processor_agent.wait_until_finished()
 
-            simple_dags = self._get_simple_dags()
+            simple_dags = self.processor_agent.harvest_simple_dags()
```

Comment:
Even one line above we access the processor agent directly.

Reply:
👍